src/AutoRest.CSharp/Common/Output/Models/Serialization/Json/JsonPropertySerialization.cs

```diff
@@ -40,6 +41,7 @@ public JsonPropertySerialization(
             ValueSerialization = valueSerialization;
             CustomSerializationMethodName = serializationHooks?.JsonSerializationMethodName;
             CustomDeserializationMethodName = serializationHooks?.JsonDeserializationMethodName;
+            DeserializeEmptyStringAsNull = property.InputModelProperty?.Decorators.Any(d => d.Name == "Azure.ClientGenerator.Core.@deserializeEmptyStringAsNull") ?? false;
```

Comment:
Could we add an extension method to `InputModelProperty` type in `Common/Input` like:
```
internal static class InputTypeExtensions
{
	public static bool ShouldDeserializeEmptyStringAsNull(this InputModelProperty property)
		=> property.Decorators.Any(d => d.Name == "Azure.ClientGenerator.Core.@deserializeEmptyStringAsNull");
}
```
and here it should be more readable

Reply:
@ArcturusZhang is it ok to resolve this?